tests/Fixer/ClassNotation/SingleTraitInsertPerStatementFixerTest.php

```diff
@@ -81,6 +81,24 @@ final class Example
 {
     use Foo, Bar ;
 }
+',
+            ],
+            'simple III' => [
+                '<?php
+class Example
+{
+    use Foo;use Bar;
```

Comment:
Should be 
```
use Foo;
use Bar;
```

not:

```
use Foo;use Bar;
```

Reply:
No, it shall not -> see the test case `simple I`.